staging/src/k8s.io/apiserver/pkg/util/shufflesharding/shufflesharding_test.go

```diff
@@ -277,7 +276,7 @@ func TestDealer_DealIntoHand(t *testing.T) {
 		},
 		{
 			"size: 6 cap: 6 slice",
-			make([]int, 6, 6),
```

Comment:
hm since this file is making assertions about the slice I think it'd be good to keep the cap set deliberately. Since without this it's not obvious that you won't get a larger cap than len.